core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java

```diff
@@ -150,23 +159,26 @@ public AuthorizeForAutoscaleV4Response authorizeForAutoscale(String crn, String
     }
 
     @Override
-    @DisableCheckPermissions
-    @PreAuthorize("hasRole('AUTOSCALE')")
+    @InternalOnly
     public CertificateV4Response getCertificate(@TenantAwareParam String crn) {
         return stackCommonService.getCertificate(crn);
     }
 
     @Override
-    @DisableCheckPermissions
-    @PreAuthorize("hasRole('AUTOSCALE')")
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
     public ClusterProxyConfiguration getClusterProxyconfiguration() {
```

Comment:
there is no crn or accountid parameter so we cannot use it for internal call, pls check readme for more knowledge: https://github.com/hortonworks/cloudbreak/tree/master/authorization-common#how-to-make-an-api-internally-callable

Reply:
Isnt it a limitation of how Internal is enabled in CB then, service to service communication without  CRN or a specific resource would always be required.
 I also tested this api it works with InternalCRN, does it mean CB bypasses all authorization checks for InternalCrn and POWERUSER_ONLY auth validation is not even invoked for internal crns ?

Reply:
yes, usually it is a limitation, since lot of service logic requires accountid for db queries, thus we need the actual account id, not the account id from internal actor CRN which is `altus` everytime.
It can happen that the logic for this does not require any db query or account id, but we usually add it, since later someone can add some query to the code
so because of these, original endpoint should be authenticated somehow, for example with this poweruseronly right, which indicates that only poweruser can call it
since we will use the new internal one, and we cannot remove old one, then this can be a solution for old endpoint, probably noone will use it

Reply:
as discussed, we will keep original endpoint, since it does not contain account level logic, and we expect that noone will break it, otherwise we can split it later also into two endpoints